package.json

```diff
@@ -1,8 +1,14 @@
 {
+  "private": true,
```

Comment:
@thomasconner this seems unintentional and looking at npm docs, this seems relevant for private repos only. Should we remove it?

Reply:
No. This prevents from publishing the SDK accidentally. Travis will deploy the SDK correctly by removing this flag.

Reply:
👍